### PR TITLE
Remove redundant code

### PR DESCRIPTION
**Pull request by ebyhr**
These codes are unused.

### CODE REVIEW

**Comment by mosabua** [comment]
Can you add in the PR or even better in the commit message what was redundant

**Comment by ebyhr** [comment]
Updated PR description.

File: gateway-ha/src/main/java/io/trino/gateway/ha/module/ClusterStateListenerModule.java [hunk 1]
```diff
@@ -53,7 +52,6 @@ public List<TrinoClusterStatsObserver> getClusterStatsObservers(
             BackendStateManager backendStateManager)
     {
         observers = new ArrayList<>();
-        NotifierConfiguration notifierConfiguration = getConfiguration().getNotifier();
```
Comment (mosabua) [inline gateway-ha/src/main/java/io/trino/gateway/ha/module/ClusterStateListenerModule.java:56]: So notifierConfiugration is never access or used? And same for all the others?

Comment (ebyhr) [reply gateway-ha/src/main/java/io/trino/gateway/ha/module/ClusterStateListenerModule.java:56]: NotifierConfiguration is still used from NotifierModule.

Comment (mosabua) [reply gateway-ha/src/main/java/io/trino/gateway/ha/module/ClusterStateListenerModule.java:56]: Sure .. but no in this class .. hence the removal. And the same logic applies to other removals.. right?

**Review by mosabua — APPROVED** [review]
Lets ship it!

**Pull request merged**